README.md

```diff
@@ -181,6 +181,7 @@ resource "pulsar_namespace" "test" {
     max_consumers_per_subscription = "50"
     max_consumers_per_topic        = "50"
     max_producers_per_topic        = "50"
+    message_ttl_seconds            = "86400"
```

Comment:
Can you alsp update the **Properties**  part in README.md?

Reply:
HI Max! The namespace_config block is not broken down anywhere in the namespace properties list. Should I create one for it? The properties are only listed in this one https://github.com/streamnative/terraform-provider-pulsar/blob/master/docs/resources/namespace.md

Reply:
Hi, Lovro!
I see, it's missing indeed. Can you take care of it? or I'll update README in future PR.

Reply:
Added a new table to the readme for the namespace config nested schema